src/server/utils/Errors.ts

```diff
@@ -29,7 +31,9 @@ export const ErrorMessage = {
   EXPECTED_QUERY_PARAM: param => `Expected query parameter not provided '${param}'`,
   INVALID_RESET: () => 'This reset request is invalid or has expired',
   USER_HAS_NO_TEAM: () => 'This user is not part of a team',
-  TEAM_FULL: (teamCode, maxSize) => `'${teamCode}' already has the maximum allowed ${maxSize} members`,
+  USER_ON_TEAM: () => `A user is already a part of the team`,
+  USER_NOT_ON_TEAM: () => `A user is not a part of the team`,
```

Comment:
is USER_NOT_ON_TEAM used anywhere? is it something for the future where we can check to make sure that all users are on a team?

Reply:
Hmm no actually, this string isn't being used anywhere